actors/builtin/miner/miner_actor.go

```diff
@@ -1673,7 +1686,9 @@ func handleProvingDeadline(rt Runtime) {
 		{
 			depositToBurn, err := st.ExpirePreCommits(store, currEpoch)
 			builtin.RequireNoErr(rt, err, exitcode.ErrIllegalState, "failed to expire pre-committed sectors")
-			penaltyTotal = big.Add(penaltyTotal, depositToBurn)
+
+			err = st.ApplyPenalty(depositToBurn)
+			builtin.RequireNoErr(rt, err, exitcode.Unwrap(err, exitcode.ErrIllegalState), "failed to apply penalty")
```

Comment:
This is never explicitly burned, just added to `FeeDebt`. The var name suggests it should be explicitly burned, but I think just applying it works?

Reply:
Yeah, PCD is decreased by the amount added to FeeDebt, so it stands to reason that this'll be burned. Nice.